src/Topology.jl

```diff
@@ -131,3 +131,26 @@ function remove_bond!(topology::Topology, i::Integer, j::Integer)
     )
     return nothing
 end
+
+function add_residue!(topology::Topology, residue::Residue)
+    check(
+        lib.chfl_topology_add_residue(topology.handle, residue.handle)
+    )
+    return nothing
+end
+
+function residues(topology::Topology)
```

Comment:
Julia have `count`, `count_zeros`, `count_ones`, `countnz`
We can rename to `count_residues`